Cliqz/React Components/Engine.swift

```diff
@@ -100,8 +100,8 @@ open class Engine {
     // MARK :- Youtube Downloader
     func findVideoLinks(url: String, callback: @escaping ([[String: Any]]) -> ()) {
         
-        let _ = self.getBridge().callAction("video-downloader:findVideoLinks", args: [url as AnyObject]) { (result) in
-            if let videoLinks = result["result"] as? [[String: Any]] {
+        let _ = self.getBridge().callAction("video-downloader:getVideoLinks", args: [url as AnyObject]) { (result) in
+            if let videoLinks = (result["result"] as! [String: Any])["formats"] as? [[String: Any]] {
```

Comment:
you should not use force unbinding (`as!`) because if this condition fails it will crash the app 
instead use the following:
`
    let results = result["result"] as? [String: Any], let videoLinks = results["formats"] as? [[String: Any]]
`

Reply:
Ok thanks! I will apply the fix 👍

Reply:
I updated as you suggested and it was fine!